server/graphql/v1/mutations/collectives.js

```diff
@@ -462,8 +463,19 @@ export async function archiveCollective(_, args, req) {
     return updatedCollective;
   }
 
-  // TODO: cascade deactivation to EVENTs and PROJECTs?
+  const projects = await collective.getProjects();
+  const events = await collective.getEvents();
 
+  if (projects && projects.length) {
+    projects.forEach(s => {
+      s.update({ isActive: false, deactivatedAt: Date.now(), approvedAt: null, HostCollectiveId: null });
```

Comment:
Looks like something we would want to `await` for, otherwise errors while archiving the children will be ignored

Reply:
you are right. Just added it